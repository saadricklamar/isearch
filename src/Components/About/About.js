import React from "react";
import { Link } from "react-router-dom";

const About = () => {
  return (
    <div className="About">
      <header className="about-header">
        <Link className="go-home" to="/">
          <i class="fas fa-arrow-left"></i>
        </Link>
      </header>
      <h2>This is an about page</h2>
    </div>
  );
};

export default About;
