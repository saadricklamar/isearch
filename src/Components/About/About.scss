.About {
  background: url(../../Assets/Mac-About-Page.jpeg) no-repeat center center
    fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  height: 100vh;
}

h2 {
  margin: 0 auto;
  color: white;
}
