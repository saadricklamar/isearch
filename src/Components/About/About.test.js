import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe("About", () => {
    let wrapper = shallow(<About/>)
    it('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})
