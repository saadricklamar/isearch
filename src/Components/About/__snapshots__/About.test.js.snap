// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match the snapshot 1`] = `ShallowWrapper {}`;
